api/src/opentrons/protocol_engine/commands/pick_up_tip.py

```diff
@@ -23,7 +23,12 @@ class PickUpTipParams(PipetteIdMixin, WellLocationMixin):
 class PickUpTipResult(BaseModel):
     """Result data from the execution of a PickUpTip."""
 
-    pass
+    # Tip volume has a default ONLY for parsing data from earlier versions, which did not include this in the result
+    tipVolume: float = Field(
+        0,
+        description="Max volume of liquid in uL for picked up tip.",
```

Comment:
Let's try to avoid abbreviations and unit approximations in description strings.
```suggestion
        description="Maximum volume of liquid that the picked up tip can hold, in µL.",
```